getting-started/campaigns.md

```diff
@@ -0,0 +1,94 @@
+- **Platforms**: Campaigns can be hosted on various platforms like CORE, Mode, or Stacks, and often involve bridging assets between blockchains to complete the tasks.
+
+## Typical Rewards in XLink Campaigns
+
+XLink campaigns offer a variety of rewards based on the campaign and the blockchain involved. Here are some common rewards you can earn:
+
+- **ALEX**: Earn ALEX by participating in campaigns on the Stacks or Ethereum blockchains.
+- **LiSTX**: A popular reward on Stacks campaigns, allowing users to stake and earn additional tokens.
+- **XLink Points**: Earn XLink Points by bridging assets and completing actions.
+- **Project-Specific Tokens**: Earn tokens tied to the specific blockchain you're interacting with (e.g., **aBTC**, **MODE Points**).
+
+## Tips for Participating in Campaigns
+
+- **Check the Requirements**: Each campaign has different requirements, so be sure to review the campaign details before you start.
+- **Stay Updated**: New campaigns are launched regularly. Keep an eye on the XLink platform or join our community channels for the latest updates.
+- **Plan Ahead**: Some campaigns offer better rewards for specific actions (e.g., staking certain tokens or adding liquidity). Plan to maximize your rewards!
+
+## Where to Find Campaigns
+
+To view and join active campaigns, visit the [**XLink Campaigns Page**](https://app.xlink.network/bridge-campaign). You’ll find detailed instructions, the supported tokens, and the rewards for each campaign.
+
+## How to Participate in XLink Campaigns
+
+Participating in campaigns is easy and rewarding. Although each campaign might be slightly different, the basic steps to participate are similar. Here’s how you can get started:
+
+### Step 1: **Register on the Platform**
+First, visit the **XLink App** and navigate to the **Campaigns** tab. This is where you'll find all the active campaigns. 
+
+Look for the **See Details** button next to the campaign you're interested in. Clicking it will provide more details on how to participate, including specific steps and links to register.
+
+![Campaign Registration](../.gitbook/assets/campaigns/campaigns-menu.png)
+
+Once you've selected a campaign, you'll need to register for it by following the instructions provided under the **How to Participate** section. You might be directed to specific platforms like **CORE**, **Mode**, or **Stacks**, depending on the campaign.
+
+For example, in the **ALEX Staking Campaign**, you'll be directed to stake your ALEX tokens on LISA. Follow the campaign-specific instructions to complete the staking process and earn rewards.
+
+![ALEX Staking Campaign Example](../.gitbook/assets/campaigns/alex-staking-example.png)
+
+For example, in the **ALEX Liquidity Campaign**, you’ll be directed to add liquidity on a supported DeFi platform. Be sure to follow the specific steps outlined in each campaign.
```

Comment:
No habria que usar de vuelta  For example..   podriamos cambiar por: 
"**Similarly,** in the **ALEX Liquidity Campaign**, you’ll be directed to add liquidity on a supported DeFi platform. Be sure to follow the specific steps outlined in each campaign."